core.js

```diff
@@ -110,18 +110,25 @@ module.exports = function (xhr) {
       // Make the request. The callback executes functions that are compatible
       // With jQuery.ajax's syntax.
       var request = options.xhr = options.xhrImplementation(ajaxSettings, function (err, resp, body) {
-          if (err || resp.statusCode >= 400) {
+          if (err) {
+              // Not sure when err !== 'undefined' && body
               if (options.error) return options.error(resp, 'error', err.message || body);
```

Comment:
This line is actually addressing #52 by returning the body if there was no clientside error. The only problem here is that err is null, so this line is the only one that needs to be fixed. I'll switch to commenting in #50 now.